.travis.yml

```diff
@@ -1,26 +1,34 @@
+cache: pip
+dist: xenial
 language: python
 python:
   - "2.7"
-  - "3.6"
+  - "3.7"
 
 env:
-  - SPHINX_SPEC=Sphinx~=1.4.0
-  - SPHINX_SPEC=Sphinx~=1.5.0
-  - SPHINX_SPEC=Sphinx~=1.6.0
-  - SPHINX_SPEC=git+https://github.com/sphinx-doc/sphinx.git\#egg\=Sphinx-dev
+  - SPHINX_SPEC='==1.4.0'
+  - SPHINX_SPEC='==1.5.0'
+  - SPHINX_SPEC='==1.6.1'
```

Comment:
What do you think about dropping support for Sphinx < 1.7? Sphinx internals have been changing at a pretty fast rate, so supporting a half a dozen versions can be tricky.

Reply:
Sounds pretty sensible to me!